Dockerfile

```diff
@@ -18,13 +18,13 @@ COPY . .
 #   docker builder prune --filter type=exec.cachemount
 RUN --mount=type=cache,target=/home/builder/.cargo,uid=$UID,gid=$GID \
 	--mount=type=cache,target=/home/builder/rust-ceramic-migration-tests/target,uid=$UID,gid=$GID \
-    make $BUILD_MODE && \
-    cp ./target/release/rust-ceramic-migration-tests ./
+    make $BUILD_MODE
 
 FROM ubuntu:latest as tester
 
 COPY --from=builder /home/builder/rust-ceramic-migration-tests/env/.env /usr/bin/.env
-COPY --from=builder /home/builder/rust-ceramic-migration-tests/rust-ceramic-migration-tests /usr/bin
+COPY --from=builder /home/builder/rust-ceramic-migration-tests/test-binaries /test-binaries
+COPY --from=builder /home/builder/rust-ceramic-migration-tests/entrypoint.sh /usr/bin/rust-ceramic-migration-tests
```

Comment:
Should the target just be called `entrypoint.sh`? Will be clearer I think.

Reply:
Agreed, updated.